jenkins/test/orchestrator/Jenkinsfile

```diff
@@ -1,44 +1,50 @@
 pipeline {
```

Comment:
Note, I would remove any changes to this file and either delete it in this PR or in a follow up PR

Reply:
Reverted the changes in https://github.com/opensearch-project/opensearch-build/pull/1429/commits/e13948505ded132571d25c0f805ae499f9dfaf46

Will fix in a follow up PR as mentioned here: https://github.com/opensearch-project/opensearch-build/pull/1429#issuecomment-1010444623